test/e2e/mongodbtests/mongodbtests.go

```diff
@@ -154,6 +174,20 @@ func ChangeVersion(mdb *mdbv1.MongoDB, newVersion string) func(*testing.T) {
 	}
 }
 
+func ChangeVersionAndFeatureCompatibilityVersion(mdb *mdbv1.MongoDB, version, fcv string) func(*testing.T) {
```

Comment:
as this is only used in the one test, maybe this could be just done in a closure in that one test. Or defined in the file of the test. Up to you 👍